samples/review-triage/pom.xml

```diff
@@ -30,7 +30,8 @@
             <version>1.0.8</version>
         </dependency>
         <dependency>
-            <groupId>org.mvnpm.at.mvnpm</groupId>
+            <groupId>org.mvnpm.at.
+            </groupId>
```

Comment:
This looks weird

Reply:
whoops - pushed from bad state. pushed the actual right commit now.